docs/make.jl

```diff
@@ -3,6 +3,7 @@ Base.HOME_PROJECT[] = abspath(Base.HOME_PROJECT[]) # JuliaLang/julia/pull/28625
 using ClimateMachine, Documenter, Literate
 
 ENV["GKSwstype"] = "100" # https://github.com/jheinen/GR.jl/issues/278#issuecomment-587090846
+ENV["CLIMATEMACHINE_SETTINGS_CUSTOM_LOGGER"] = "OFF" # to avoid clashing logger with Documenter/Literate
```

Comment:
to be consistent we should probably stick with `"true"` / `"false"` (seems like this is widely used in the Julia world as it's easy to parse).

Reply:
Why not just use Boolean, then? @simonbyrne should we switch to Bool?

@jakebolewski any idea why this is failing?
```
ERROR: LoadError: LoadError: LoadError: MethodError: no method matching tryparse(::Type{String}, ::String)
Closest candidates are:
  tryparse(!Matched::Type{Float64}, ::String) at parse.jl:245
  tryparse(!Matched::Type{Float32}, ::String) at parse.jl:265
  tryparse(!Matched::Type{Float16}, ::AbstractString) at parse.jl:285
  ...
in expression starting at string:1
when executing the following code block

```

Reply:
there doesn't seem to be a pass-through method defined for strings :(

Reply:
I would just switch to bool, I don't think we use ON / OFF anywhere else